Bindings/Laerdal.Maui.AzureCommunicationCalling.iOS/create_binding.sh

```diff
@@ -11,14 +11,24 @@ cd ..
 
 # Now make a fat framework containing both x86 and arm64 binaries
```

Comment:
This is mostly on me I guess, this script assumes that you downloaded the release zip file and unzipped in correct folder?

Reply:
In current form everything will be downloaded automatically by the script.
You don't have to download/unzip anything.